### PR TITLE
[BREAKING] feat: migrating from generators to async/await

### PR DESCRIPTION
**Pull request by gxcsoccer**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by gxcsoccer** [comment]
https://github.com/node-modules/cluster-client/issues/37

@XadillaX

File: lib/utils.js [hunk 1]
```diff
@@ -70,15 +71,17 @@ exports.formatKey = formatKey;
...
+exports.callFn = async function(fn, args) {
   args = args || [];
   if (!is.function(fn)) return;
   if (is.generatorFunction(fn)) {
-    return yield fn(...args);
+    return await co(function* () {
+      return yield fn(...args);
+    });
   }
   const r = fn(...args);
```
Comment (popomore) [inline lib/utils.js]: 好像可以直接 await

**Comment by popomore** [comment]
这个最好用 `feat:`，要发大版本了

File: lib/utils.js (hunk 1 above)
Comment (gxcsoccer) [reply lib/utils.js]: 哪里？

Comment (popomore) [reply lib/utils.js]: `await fn(...args);`

**Comment by gxcsoccer** [comment]
2.0.0

**Pull request merged**